src/legacy/status_im/communities/e2e.cljs

```diff
@@ -1,35 +1,196 @@
 (ns legacy.status-im.communities.e2e
-  (:require [taoensso.timbre :as log]
+  (:require [promesa.core :as p]
+            [status-im.common.json-rpc.events :as rpc]
+            [status-im.constants :as constants]
+            [taoensso.timbre :as log]
             [utils.re-frame :as rf]))
 
 ;;NOTE: ONLY FOR QA
 
-(rf/defn create-closed-community
-  {:events [:fast-create-community/create-closed-community]}
+(def one-stt-in-wei "1000000000000000000")
+(def one-eth-in-wei "1000000000000000000")
+(def ten-stt-in-wei "10000000000000000000")
+
+(def stt-symbol "STT")
+(def eth-symbol "ETH")
+(def null-address "0x0000000000000000000000000000000000000000")
+
+(defn eth-token-criteria
+  [amount-in-wei]
+  {:contractAddresses {constants/ethereum-sepolia-chain-id null-address
+                       constants/arbitrum-sepolia-chain-id null-address
+                       constants/optimism-sepolia-chain-id null-address}
+   :type              constants/community-token-type-erc20
+   :symbol            "ETH"
+   :name              "Ethereum"
+   :amountInWei       amount-in-wei
+   :decimals          18})
+
+(defn stt-token-criteria
+  [amount-in-wei]
+  {:contractAddresses {constants/ethereum-sepolia-chain-id constants/sepolia-stt-contract-address}
+   :type              constants/community-token-type-erc20
+   :symbol            "STT"
+   :name              "Status Test Token"
+   :amountInWei       amount-in-wei
+   :decimals          18})
+
+(def token-criteria
+  {stt-symbol stt-token-criteria
+   eth-symbol eth-token-criteria})
+
+(def base-channel-permissions
+  [{:permission-type constants/community-token-permission-can-view-channel
+    :token-symbol    stt-symbol
+    :amount          one-stt-in-wei}
+   {:permission-type constants/community-token-permission-can-view-channel
+    :token-symbol    eth-symbol
+    :amount          one-eth-in-wei}
+   {:permission-type constants/community-token-permission-can-view-and-post-channel
+    :token-symbol    stt-symbol
+    :amount          one-stt-in-wei}
+   {:permission-type constants/community-token-permission-can-view-and-post-channel
+    :token-symbol    eth-symbol
+    :amount          one-eth-in-wei}])
+
+(defn channel-names->description
+  [channel-names]
+  (map #(hash-map :name        %1
+                  :permissions [%2])
+       channel-names
+       base-channel-permissions))
+
+(def community-descriptions
```

Comment:
can we have some icons for communities as well? We're using them during development as well and will make it easier to test UI issues. Or if these are used mainly for e2e, will be nice to have an issue to create & customize communities quickly during development.

Reply:
@clauxx i believe there is a difference between creating communities on desktop and mobile it's encouraged to use the desktop to create them for development even if it adds a bit more friction to the process.

Reply:
I think the only thing I can do, is to have something that distinguish them if you create multiple, as that was useful for development (anything, the name will be the same as qa selects them by name, but say a random color), but yes, as Jamie pointed out, no extra options etc, but you can always modify this namespace manually during development to create a suitable community (it's all rpc requests)